omise/classes/payment_order.php

```diff
@@ -0,0 +1,117 @@
+<?php
+if (! defined('_PS_VERSION_')) {
+    exit();
+}
+
+class PaymentOrder extends ModuleFrontController
+{
+    protected function getCartId()
```

Comment:
@nimid how about just have method 

```php
protected function cart()
{
    return $this->context->cart;
}
```

Then you just call 
```php
$this->cart()->id;
$this->cart()->id_customer;
$this->cart()->getOrderTotal();
```

or even you can call `$this->context->cart->id` directly without providing getter methods.